components/ActualGamesList/ActualGamesList.tsx

```diff
@@ -2,8 +2,20 @@ import { useTranslation } from 'react-i18next';
 import { ActualGame } from './ActualGame/ActualGame';
 import { ActualGamesListPropsI } from './ActualGamesListProps';
 import styles from './ActualGamesList.module.scss';
+import { getRulesContract } from 'gameApi';
+import { useState } from 'react';
 export const ActualGamesList: React.FC<ActualGamesListPropsI> = ({ gamesList, onClick }) => {
   const { t } = useTranslation();
+
+  let [ticTacToeAddress, setTicTacToeAddress] = useState('')
+  let [checkersAddress, setCheckersAddress] = useState('')
```

Comment:
Are there any other components that use this data, except the ActualGame? 
Are there any reasons to put these outside the component that uses them?